src/main/java/org/folio/circulation/domain/OpeningPeriod.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+public class OpeningPeriod {
+  private static final String OPENING_DAY_KEY = "openingDay";
+  private static final String DATE_KEY = "date";
+  private static final String DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
+    DateTimeFormat.forPattern(DATE_TIME_FORMAT).withZoneUTC();
+
+  private final OpeningDay openingDay;
+  private final LocalDate date;
+
+  public OpeningDay getOpeningDay() {
+    return openingDay;
+  }
+
+  public LocalDate getDate() {
+    return date;
+  }
+
+  public OpeningPeriod(LocalDate date, OpeningDay openingDay) {
+    this.openingDay = openingDay;
+    this.date = date;
+  }
+
+  public OpeningPeriod(JsonObject jsonObject) {
+    this.openingDay = OpeningDay.fromJsonByDefaultKey(jsonObject);
+    String dateProperty = jsonObject.getString(DATE_KEY);
+    this.date = LocalDate.parse(dateProperty, DATE_TIME_FORMATTER);
```

Comment:
`DateTime.parse(dateProperty)` should also work for the date time is ISO-8601 standard.

Reply:
it does, fixed